From baf7ab915d139d68b6ab332759c991be36967622 Mon Sep 17 00:00:00 2001
From: "G.h. Gao" <guanhua.gao@nxp.com>
Date: Thu, 7 Mar 2019 10:04:20 +0800
Subject: [PATCH 068/235] dpaa2-eth: Fix compiling errors

Fix compiling errors introduced by removing queue_id.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
index 6a3ff1637d9b..30ca0918341a 100755
--- a/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
+++ b/drivers/staging/fsl-dpaa2/ethernet/dpaa2-eth.c
@@ -389,7 +389,7 @@ static void dpaa2_eth_rx(struct dpaa2_eth_priv *priv,
 
 	if (fd_format == dpaa2_fd_single) {
 		xdp_act = dpaa2_eth_run_xdp(priv, ch, (struct dpaa2_fd *)fd,
-					    queue_id, vaddr);
+					    fq->flowid, vaddr);
 		if (xdp_act != XDP_PASS)
 			return;
 
@@ -938,7 +938,7 @@ static void dpaa2_eth_tx_conf(struct dpaa2_eth_priv *priv,
 	fq->dq_bytes += fd_len;
 
 	/* Check congestion state and wake all queues if necessary */
-	if (unlikely(__netif_subqueue_stopped(priv->net_dev, queue_id))) {
+	if (unlikely(__netif_subqueue_stopped(priv->net_dev, fq->flowid))) {
 		dma_sync_single_for_cpu(dev, priv->cscn_dma,
 					DPAA2_CSCN_SIZE, DMA_FROM_DEVICE);
 		if (!dpaa2_cscn_state_congested(priv->cscn_mem))
-- 
2.17.1

